components/textInputField/TextInputField.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import styled from 'styled-components';
+import { min } from '../../lib/responsive';
+import { fonts, P, fontWeights } from '../../style/typography';
+import { colors } from '../../style/colors';
+
+export const TextLabel = styled(P)`
+  color: ${colors.BROWN};
+  display: block;
+  margin: 0;
+`;
+
+// here I want to reuse the P for lines 15 - 20 but it needs
+// to stay as an input for functinality
+export const TextInput = styled.input`
```

Comment:
Could try this?
https://stackoverflow.com/questions/55776961/styled-components-extend-styles-and-change-element-type